pom.xml

```diff
@@ -61,7 +61,7 @@
         <dependency>
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-ext</artifactId>
-            <version>1.0.0</version>
+            <version>1.2.0</version>
```

Comment:
Since Apache Guacamole 1.2.0 is compatible with extensions written for any older version of 1.x, I think we should stick with the minimum API version required for the features implemented. For the new version of `connect()` that receives tokens, that would be 1.1.0.